api/src/main/java/jakarta/data/constraint/In.java

```diff
@@ -27,8 +29,66 @@
 
 import static java.util.Collections.unmodifiableList;
 
+/**
+ * <p>A constraint that requires equality with one member of a set.</p>
```

Comment:
I am not sure if `one member of a set` 

I mean, it would be great if we define it as one or more items represented by a Set...

Reply:
Equality with 2 members of a set is impossible because a set cannot have 2 members that are equal to each other.  I should have used the term list or collection instead.  I'll fix it.

Reply:
Corrected under https://github.com/jakartaee/data/pull/1176/commits/f60451d0a787e01bee30e599cb6cbe5464259864